app/service/ingest.py

```diff
@@ -28,6 +31,8 @@
     IngestFamilyDTO,
 )
 
+DOCUMENT_INGEST_LIMIT = 1000
```

Comment:
please can we make this a parameter - so we can pass in via the cli ... we will need to alter this ... not an issue for this PR so let's raise a ticket

Reply:
Good call. And actually this will make testing easier. I'll raise the ticket